tcav/results.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto2";
+
```

Comment:
i imagine for this code to work, you need the compiled py file for this proto as well. should include it in here, or have it be compiled automatically and put with these other files when the pip package is created.

Reply:
I'm unfamiliar with compiling protos outside of Blaze Build - is there a reference on how I can generate a compiled proto py file?

Reply:
See https://developers.google.com/protocol-buffers/docs/pythontutorial section on compiling

Reply:
done